fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -563,11 +569,31 @@ public void replaceProperties(final IdentifierConverter<Resource, FedoraResource
             exceptions.append(e.getMessage());
         }
 
+        removeEmptyFragments();
+
         if (exceptions.length() > 0) {
             throw new MalformedRdfException(exceptions.toString());
         }
     }
 
+    private void removeEmptyFragments() {
+        try {
+            if (node.hasNode("#")) {
```

Comment:
If there isn't a "#" node, then this will fail.  I tried removing it, and there are about a dozen IT failures, so I think we need to keep it.